frontend/.env.sample

```diff
@@ -1,5 +1,5 @@
-NEXT_PUBLIC_API_URL=""
-NEXTAUTH_URL=http://localhost:3000
+NEXT_PUBLIC_DOMAIN=https://staging.heco.vizzuality.com
+NEXT_PUBLIC_BACKEND_DOMAIN=https://staging.heco.vizzuality.com
+NEXT_PUBLIC_API_PATH=/backend/api/v1
```

Comment:
I would strongly encourage you to refactor your code, so that this value has `/backend` in it, and everything else is part of the endpoint subpath, not the baseUrl:
- /api is part of the method signature, not the URL of the API. The backend can choose to implement something at https://staging.heco.vizzuality.com/backend/auth (for example) which is valid and would break your implementation
- /v1 is evil because if the api decides to add a /v2 set of endpoints, you now have to add yet another variable.